docs/src/man/importing_and_exporting.md

```diff
@@ -1,5 +1,24 @@
 # Importing and Exporting Data (I/O)
 
+## JLD2 Files
+
+A convenient format for reading and writing DataFrames in JLD2 which saves and loads
+Julia data structures in a format comprising a subset of HDF5. Unlike several other 
+formats, JLD2 preserves custom Types. The save and load functions, provided by FileIO, 
+provide a mechanism to read/write `DataFrame` from/to a JLD2 file.  
+
+A `DataFrame` can be saved as a JLD2 file at path `output.jld2` using
+```julia
+using FileIO
```

Comment:
I would assume it also requres JLD2.jl package loaded - right?

Reply:
JLD2 now added